index.js

```diff
@@ -77,73 +77,68 @@ async function uploadArtifact(artifactClient, artifactName, artifactPath,retenti
 
 function findGitFolder(startPath) {
 
-    try
-    {
-      if (!fs.existsSync(startPath)) {
-          return null;
-      }
+  try {
+    if (!fs.existsSync(startPath)) {
+      return null;
+    }
 
-      const files = fs.readdirSync(startPath);
+    const files = fs.readdirSync(startPath);
 
-      for (let i = 0; i < files.length; i++) {
-          const filePath = path.join(startPath, files[i]);
+    for (let i = 0; i < files.length; i++) {
+      const filePath = path.join(startPath, files[i]);
 
-          if (files[i] === '.git' && fs.statSync(filePath).isDirectory()) {
-              return filePath;
-          }
+      if (files[i] === '.git' && fs.statSync(filePath).isDirectory()) {
+        return filePath;
+      }
 
-          if (fs.statSync(filePath).isDirectory()) {
-              const result = findGitFolder(filePath);
-              if (result) {
-                  return result;
-              }
-          }
+      if (fs.statSync(filePath).isDirectory()) {
+        const result = findGitFolder(filePath);
+        if (result) {
+          return result;
+        }
       }
     }
-    catch (exceptionVar) {
-      console.log(exceptionVar)
-    }
+  }
+  catch (exceptionVar) {
+    console.log(exceptionVar)
+  }
 
-    return null;
+  return null;
 }
 
 function hasGitFolderWithGitHubRunnerToken(pathToCheck) {
-  const fs = require('fs');
-  const path = require('path');
-  try
-  {
-  const gitDir = findGitFolder(pathToCheck, '.git');
-  if (gitDir) {
-  const configFile = path.join(gitDir, 'config');
-  const regex = new RegExp('eC1hY2Nlc3MtdG9rZW46Z2hz', 'i');
-
   try {
-    if (fs.existsSync(gitDir) && fs.existsSync(configFile)) {
-      const configContent = fs.readFileSync(configFile, 'utf-8');
-      if (regex.test(configContent)) {      
-          return configFile;
+    const gitDir = findGitFolder(pathToCheck, '.git');
+    if (gitDir) {
+    const configFile = path.join(gitDir, 'config');
+    const regex = new RegExp('eC1hY2Nlc3MtdG9rZW46Z2hz', 'i');
+
+    try {
+      if (fs.existsSync(gitDir) && fs.existsSync(configFile)) {
+        const configContent = fs.readFileSync(configFile, 'utf-8');
+        if (regex.test(configContent)) {
+            return configFile;
+        }
+      }
+      } catch (error) {
+        console.error('Error checking Git config:', error);
+        return null;
       }
     }
-    } catch (error) {
-      console.error('Error checking Git config:', error);
-      return null;
-    }
-  }
   } catch (err) {
     console.log(err)
   }
 }
 
-async function populateFilesWithFullPath(rootPath,includeHiddenFiles) {
-  const fs = require('fs').promises; // Use promises for cleaner async/await usage
```

Comment:
This is the only place that doesn't use `fs` to mean `fs`. So, this changes it to `fsPromises` to make it clear.